src/js/components/Overview/PublicOverview.tsx

```diff
@@ -94,7 +94,7 @@ const PublicOverview = () => {
         <Col flex={1}>
           {displayedSections.map(({ sectionTitle, charts }, i) => (
             <div key={i} className="overview">
-              <OverviewSection title={sectionTitle} chartData={charts} />
+              <OverviewSection title={sectionTitle} chartData={charts} searchableFields={searchableFields} />
```

Comment:
Feels like an unnecessary prop drill. I would suggest searchableFields to be an object with keys as sections and values as the searchable fields in that section. I do understand that calling it here reduces to calls to useSearchableFields but also we are passing irrelevant data to sections.

Reply:
i agree the prop drilling is a bit much. was avoiding multiple hook calls but i doubt the performance hit is that bad.

Reply:
actually, since query sections != overview sections, we do need to pass everything